editor/store-defaults.js

```diff
@@ -1,7 +1,9 @@
+import { viewPort } from '../utils';
```

Comment:
We shouldn't traverse into another top-level directory like this, and should instead treat is as a "WordPress dependency", i.e.:

```js
/**
 * WordPress dependencies
 */
import { viewPort } from '@wordpress/utils';
```